src/aat/resources/features/F-045/F-045.feature

```diff
@@ -0,0 +1,65 @@
+@F-045
+Feature: Grant access to case
+
+  Background:
+    Given an appropriate test context as detailed in the test data source
+
+  @S-151
+  Scenario: must return 201 if the grant is successful for a user to a valid case ID
+    Given a user with [a detailed profile in CCD]
+    When a request is prepared with appropriate values
+    And the request [has a valid case-reference in input parameters]
```

Comment:
a valid case-reference existing in CCD ...

Reply:
updated